src/applications/disability-benefits/new-686/config/chapters/veteran-information/veteran-address/veteran-address.js

```diff
@@ -47,6 +53,9 @@ export const uiSchema = {
     },
     'ui:required': () => true,
     'ui:title': 'Phone Number',
+    'ui:errorMessages': {
+      pattern: 'Please enter only numbers, no dashes or parenthesis',
```

Comment:
I think it should be parentheses instead of parenthesis..[TIL](https://english.stackexchange.com/questions/130219/parentheses-vs-parenthesis)

Reply:
fixed